src/main/scala/nl.knaw.dans.easy.managedeposit/EasyManageDepositApp.scala

```diff
@@ -301,7 +312,9 @@ class EasyManageDepositApp(configuration: Configuration) extends DebugEnhancedLo
         deposit.depositor,
         deposit.depositId,
         deposit.state,
-        deposit.doi.getOrElse("n/a"),
+        deposit.identifier.doi.value.getOrElse("n/a"),
+        deposit.identifier.doi.registeredString.getOrElse("yes"),
```

Comment:
Not totally sure if this is a good change, but wouldn't it be nicer to do something like `.getOrElse(true)`, rather than construct the `String` yourself. Then `BooleanUtils` can do all the `Boolean -> String` conversions.

In this cause, however, you cannot use `.registeredString`, as you currently do. But maybe you can come up with something in between?

```suggestion
        BooleanUtils.toStringYesNo(deposit.identifier.doi.registered.getOrElse(true)),
```